test/org/sagebionetworks/bridge/services/StudyServiceMockTest.java

```diff
@@ -631,6 +631,16 @@ public void loadingStudyWithoutAccountExistsSmsTemplateAddsADefault() {
         assertNotNull(retStudy.getAccountExistsSmsTemplate());
     }
     
+    @Test
+    public void loadingStudyWithoutSignedConsentSmsTemplateAddsADefault() {
+        Study study = TestUtils.getValidStudy(StudyServiceMockTest.class);
+        study.setSignedConsentSmsTemplate(null);
+        when(studyDao.getStudy("foo")).thenReturn(study);
+        
+        Study retStudy = service.getStudy("foo");
+        assertNotNull(retStudy.getAccountExistsSmsTemplate());
```

Comment:
Copy-paste error. This should be getSignedConsentSmsTemplate(), not getAccountExistsSmsTemplte(). (Here and in updateStudyWithoutSignedConsentSmsTemplateAddsADefault() and in createStudyWithoutSignedConsentSmsTemplateAddsADefault().

Reply:
Ack! fixed these